hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -1720,7 +1740,9 @@ HapiSpec payTestSelfDestructCall() {
                         cryptoCreate(PAYER).balance(1_000_000_000_000L).logged(),
                         cryptoCreate(RECEIVER).balance(1_000L),
                         uploadInitCode(contract),
-                        contractCreate(contract))
+                    contractCreate(contract),
+                    getContractInfo(contract)
+                        .exposingEvmAddress(cb -> tokenCreateContractAddress.set(asHeadlongAddress(cb))))
```

Comment:
Getting the contract id dynamically because when we convert the test to use eth context it doesn't find the contract address id